src/components/Heading/Heading.tsx

```diff
@@ -6,14 +6,28 @@ import {
 import React, { forwardRef } from "react";
 
 import Link from "../Link/Link";
+import useNYPLBreakpoints from "../../hooks/useNYPLBreakpoints";
 
 export const headingSizesArray = [
+  "display1",
```

Comment:
What will/should display1 be used for?

Reply:
Based on your later comments, I think you found the description on the `Typography` style guide page.

Reply:
I am not 100% stuck on using these values, but I do like them. That being said, once I push my updates, take a look at the language I added to the docs. See if that makes it feel any better.

Reply:
I am not 100% stuck on using these values, but I do like them. That being said, once I push my updates, take a look at the language I added to the docs. See if that makes it feel any better.